types/types.go

```diff
@@ -129,7 +129,7 @@ type NodeConfig struct {
 	// Bind mounts strings (src:dest:options).
 	Binds []string `json:"binds,omitempty"`
 	// PortBindings define the bindings between the container ports and host ports
-	PortBindings nat.PortMap `json:"portbindings,omitempty"`
+	PortBindings nat.PortMap `json:"-"`
```

Comment:
was this removed for testing?

Reply:
or you removed it because portbindings is now substituted by `port-bindings` which is calculated based on runtime state?

Reply:
Yes the latter was my thinking. The assumption is, that you're interested in the actual running state, which is a concret port per mapping and the possible range is pretty much of no interest...

Reply:
https://github.com/srl-labs/containerlab/pull/1430/files#diff-bb9cd8f6a91720041aaf633ac5fbaf759782c31c70128e050bc1ef0d027dd755R46